README.md

```diff
@@ -1,13 +1,32 @@
 [![Build Status](https://travis-ci.org/CICE-Consortium/CICE.svg?branch=master)](https://travis-ci.org/CICE-Consortium/CICE)
 [![Documentation Status](https://readthedocs.org/projects/cice-consortium-cice/badge/?version=master)](http://cice-consortium-cice.readthedocs.io/en/master/?badge=master)
 
-## Overview
-This repository contains the files and code needed to run the CICE sea ice numerical model starting with version 6. CICE is maintained by the CICE Consortium. Versions prior to v6 are found in the [CICE-svn-trunk repository](https://github.com/CICE-Consortium/CICE-svn-trunk).
+## The CICE Consortium sea-ice model
```

Comment:
@eclare108213 may want to weigh into how we refer to this model. My inclination would be to label this section "The CICE sea-ice model"

Reply:
I agree it's more natural to just say "CICE sea-ice model"

Reply:
I've left this as 'The CICE Consortium sea-ice model' for now, but we can change it if others think that sounds weird.